response.go

```diff
@@ -159,6 +160,14 @@ func getResponseContent(opChain *chain, resp *http.Response) []byte {
 	return content
 }
 
+func (r *Response) getContent(opChain *chain) []byte {
+	if !r.hasRead {
+		r.content = getResponseContent(opChain, r.httpResp)
```

Comment:
getResponseContent is invoked only from getContent, and getContent is very small. So I suggest to merge getResponseContent into getContent.